src/Microsoft.AspNet.SignalR.Core/Hubs/Lookup/ReflectedHubDescriptorProvider.cs

```diff
@@ -11,133 +11,132 @@
-                            Resources.Error_DuplicateHubNames,
-                            oldDescriptor.HubType.AssemblyQualifiedName,
-                            descriptor.HubType.AssemblyQualifiedName,
-                            descriptor.Name));
-                }
-            }
-
-            return cacheEntries;
-        }
-
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes", Justification = "If we throw then it's not a hub type")]
-        private static bool IsHubType(Type type)
-        {
-            try
-            {
-                return typeof(IHub).IsAssignableFrom(type) &&
-                       !type.IsAbstract &&
-                       (type.Attributes.HasFlag(TypeAttributes.Public) ||
-                        type.Attributes.HasFlag(TypeAttributes.NestedPublic));
-            }
-            catch
-            {
-                return false;
-            }
-        }
-
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes", Justification = "If we throw then we have an empty type")]
-        private IEnumerable<Type> GetTypesSafe(Assembly a)
-        {
-            try
-            {
-                return a.GetTypes();
-            }
-            catch (ReflectionTypeLoadException ex)
-            {
-                _trace.TraceWarning("Some of the classes from assembly \"{0}\" could Not be loaded when searching for Hubs. [{1}]\r\n" +
-                                    "Original exception type: {2}\r\n" +
-                                    "Original exception message: {3}\r\n",
-                                    a.FullName,
-                                    a.Location,
-                                    ex.GetType().Name,
-                                    ex.Message);
-
-                if (ex.LoaderExceptions != null)
-                {
-                    _trace.TraceWarning("Loader exceptions messages: ");
-
-                    foreach (var exception in ex.LoaderExceptions)
-                    {
-                        _trace.TraceWarning("{0}\r\n", exception);
-                    }
-                }
-
-                return ex.Types.Where(t => t != null);
-            }
-            catch (Exception ex)
-            {
-                _trace.TraceWarning("None of the classes from assembly \"{0}\" could be loaded when searching for Hubs. [{1}]\r\n" +
-                                    "Original exception type: {2}\r\n" +
-                                    "Original exception message: {3}\r\n",
-                                    a.FullName,
-                                    a.Location,
-                                    ex.GetType().Name,
-                                    ex.Message);
-
-                return Enumerable.Empty<Type>();
-            }
-        }
-    }
+	public class ReflectedHubDescriptorProvider : IHubDescriptorProvider
```

Comment:
For future reference, please avoid major reformatting like this in your contributions. We have to reformat it back to our standard code style in order to merge it :). Don't worry about it for this change, I can review the actual functional change just fine and resolve the formatting when I merge, but just for the future :).

Reply:
I have noticed that only after pushed the changes. I have reworked this during rebase.